### DISCUSSION TITLE
Receiving router/page context in reportWebVitals

### DISCUSSION THREAD

**Discussion by arunesh90** [vercel/next.js#16243] [Help] [answered]
Hello 👋 

Does someone happen to know if it's possible to receive the current router/page context in the [`reportWebVitals`](https://nextjs.org/docs/advanced-features/measuring-performance) function that you can set in `_app`?

After some quick glancing and some attempts, it seems like you're currently only limited to getting the web vitals data with no additional context attached to it. I think it would be beneficial to also have additional (page/router) context, so it's easier to keep track of pages that could need some more tweaking.

**Comment by einSelbst** [comment, marked as answer]
Old question, however, this newer question about the same got the answer that this is planned and mentions a workaround https://github.com/vercel/next.js/discussions/18614

**Reply by arunesh90** [reply]
Thanks for letting me know 🙂
